tasks/main.yml

```diff
@@ -47,15 +47,30 @@
     id: "6ABA09963E518866C29FA6C10F0921E4A2D0D6AE"
     state: absent
 
+- name: Remove bearwall2 repository entry without signed-by
+  apt_repository:
+    repo: "deb https://dl.cloudsmith.io/public/bearwall/bearwall2/deb/{{ ansible_distribution | lower }} {{ ansible_distribution_release }} main"
+    state: absent
+
+- name: Remove bearwall2 cloudsmith GPG signing key from default keyring
+  apt_key:
+    id: "{{ bearwall2_apt_key_fingerprint }}"
+    keyring: "/etc/apt/trusted.gpg"
+    state: absent
+
 - name: Import bearwall2 cloudsmith GPG signing key
   apt_key:
-    id: "C704B6DE77346EBAE0D229DA26EA35E7A5ED1FFE"
+    id: "{{ bearwall2_apt_key_fingerprint }}"
     url: "https://dl.cloudsmith.io/public/bearwall/bearwall2/gpg.26EA35E7A5ED1FFE.key"
+    keyring: "/usr/share/keyrings/bearwall2.gpg"
     state: present
 
 - name: Add bearwall2 cloudsmith repository
   apt_repository:
```

Comment:
Can we add a `filename: bearwall2` to this task so that we don't get the default generated filename.